Patch to use Aliens bin path instead of one inside /P/Ruby,
so that executables from gems don't try to write into the /P tree.
--- ruby-1.9.2-p290/lib/rubygems.rb.orig	2017-07-08 01:29:10.809269886 -0300
+++ ruby-1.9.2-p290/lib/rubygems.rb	2017-07-08 01:29:14.121269948 -0300
@@ -131,13 +131,17 @@ module Gem
     :EXEEXT            => RbConfig::CONFIG["EXEEXT"],
     :RUBY_SO_NAME      => RbConfig::CONFIG["RUBY_SO_NAME"],
     :arch              => RbConfig::CONFIG["arch"],
-    :bindir            => RbConfig::CONFIG["bindir"],
-    :datadir           => RbConfig::CONFIG["datadir"],
-    :libdir            => RbConfig::CONFIG["libdir"],
+    :bindir  => File.join('@%Compile_goboSystem%@/Aliens/RubyGems/',
+                         ConfigMap[:ruby_version], 'bin'),
+    :datadir => File.join('@%Compile_goboSystem%@/Aliens/RubyGems/',
+                         ConfigMap[:ruby_version], 'share'),
+    :libdir  => File.join('@%Compile_goboSystem%@/Aliens/RubyGems/',
+                         ConfigMap[:ruby_version], 'lib'),
     :ruby_install_name => RbConfig::CONFIG["ruby_install_name"],
     :ruby_version      => RbConfig::CONFIG["ruby_version"],
     :rubylibprefix     => RbConfig::CONFIG["rubylibprefix"],
-    :sitedir           => RbConfig::CONFIG["sitedir"],
+    :sitedir => File.join('@%Compile_goboSystem%@/Aliens/RubyGems/',
+                         ConfigMap[:ruby_version], 'lib/site_ruby'),
     :sitelibdir        => RbConfig::CONFIG["sitelibdir"],
     :vendordir         => RbConfig::CONFIG["vendordir"] ,
     :vendorlibdir      => RbConfig::CONFIG["vendorlibdir"]
